Upstream: yes
Source: https://github.com/pkgconf/pkgconf/commit/d67a04011f92f4eef48dd92a3306fb61163cee06
(converted to binary diff because it made patch choke on the dos-lineendings test)

diff --binary -x .git/ -u -r a/tests/lib1/argv-parse-2.pc b/tests/lib1/argv-parse-2.pc
--- a/tests/lib1/argv-parse-2.pc	2015-07-20 04:16:25.390552607 -0400
+++ b/tests/lib1/argv-parse-2.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test
+prefix=/usr
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib1/argv-parse-3.pc b/tests/lib1/argv-parse-3.pc
--- a/tests/lib1/argv-parse-3.pc	2015-07-20 04:16:25.390552607 -0400
+++ b/tests/lib1/argv-parse-3.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test
+prefix=/usr
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib1/argv-parse.pc b/tests/lib1/argv-parse.pc
--- a/tests/lib1/argv-parse.pc	2015-07-20 04:16:25.390552607 -0400
+++ b/tests/lib1/argv-parse.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test
+prefix=/usr
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib1/bar.pc b/tests/lib1/bar.pc
--- a/tests/lib1/bar.pc	2015-07-20 04:16:25.390552607 -0400
+++ b/tests/lib1/bar.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test
+prefix=/usr
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib1/baz.pc b/tests/lib1/baz.pc
--- a/tests/lib1/baz.pc	2015-07-20 04:16:25.390552607 -0400
+++ b/tests/lib1/baz.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test
+prefix=/usr
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib1/cflags-libs-only.pc b/tests/lib1/cflags-libs-only.pc
--- a/tests/lib1/cflags-libs-only.pc	2015-07-20 04:16:25.390552607 -0400
+++ b/tests/lib1/cflags-libs-only.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test/local
+prefix=/usr/local
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib1/circular-1.pc b/tests/lib1/circular-1.pc
--- a/tests/lib1/circular-1.pc	2015-07-20 04:16:25.391553107 -0400
+++ b/tests/lib1/circular-1.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test
+prefix=/usr
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib1/circular-2.pc b/tests/lib1/circular-2.pc
--- a/tests/lib1/circular-2.pc	2015-07-20 04:16:25.391553107 -0400
+++ b/tests/lib1/circular-2.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test
+prefix=/usr
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib1/circular-3.pc b/tests/lib1/circular-3.pc
--- a/tests/lib1/circular-3.pc	2015-07-20 04:16:25.391553107 -0400
+++ b/tests/lib1/circular-3.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test
+prefix=/usr
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib1/conflicts.pc b/tests/lib1/conflicts.pc
--- a/tests/lib1/conflicts.pc	2015-07-20 04:16:25.391553107 -0400
+++ b/tests/lib1/conflicts.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test
+prefix=/usr
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib1/dos-lineendings.pc b/tests/lib1/dos-lineendings.pc
--- a/tests/lib1/dos-lineendings.pc	2015-07-20 04:16:25.391553107 -0400
+++ b/tests/lib1/dos-lineendings.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test
+prefix=/usr
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib1/foo.pc b/tests/lib1/foo.pc
--- a/tests/lib1/foo.pc	2015-07-20 04:16:25.391553107 -0400
+++ b/tests/lib1/foo.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test
+prefix=/usr
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib1/framework-1.pc b/tests/lib1/framework-1.pc
--- a/tests/lib1/framework-1.pc	2015-07-20 04:16:25.391553107 -0400
+++ b/tests/lib1/framework-1.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test
+prefix=/usr
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib1/framework-2.pc b/tests/lib1/framework-2.pc
--- a/tests/lib1/framework-2.pc	2015-07-20 04:16:25.392553607 -0400
+++ b/tests/lib1/framework-2.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test
+prefix=/usr
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib1/intermediary-1.pc b/tests/lib1/intermediary-1.pc
--- a/tests/lib1/intermediary-1.pc	2015-07-20 04:16:25.392553607 -0400
+++ b/tests/lib1/intermediary-1.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test
+prefix=/usr
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib1/intermediary-2.pc b/tests/lib1/intermediary-2.pc
--- a/tests/lib1/intermediary-2.pc	2015-07-20 04:16:25.392553607 -0400
+++ b/tests/lib1/intermediary-2.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test
+prefix=/usr
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib1/missing-require.pc b/tests/lib1/missing-require.pc
--- a/tests/lib1/missing-require.pc	2015-07-20 04:16:25.392553607 -0400
+++ b/tests/lib1/missing-require.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test
+prefix=/usr
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib1/multiline.pc b/tests/lib1/multiline.pc
--- a/tests/lib1/multiline.pc	2015-07-20 04:16:25.392553607 -0400
+++ b/tests/lib1/multiline.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test
+prefix=/usr
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib1/nocflag.pc b/tests/lib1/nocflag.pc
--- a/tests/lib1/nocflag.pc	2015-07-20 04:16:25.392553607 -0400
+++ b/tests/lib1/nocflag.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test
+prefix=/usr
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib1/nolib.pc b/tests/lib1/nolib.pc
--- a/tests/lib1/nolib.pc	2015-07-20 04:16:25.393554107 -0400
+++ b/tests/lib1/nolib.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test
+prefix=/usr
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib1/no-trailing-newline.pc b/tests/lib1/no-trailing-newline.pc
--- a/tests/lib1/no-trailing-newline.pc	2015-07-20 04:16:25.392553607 -0400
+++ b/tests/lib1/no-trailing-newline.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test
+prefix=/usr
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
@@ -7,4 +7,4 @@
 Description: A testing pkg-config file
 Version: 1.2.3
 Libs: -L${libdir}/no-trailing-newline -lno-trailing-newline
-Cflags: -I${includedir}/no-trailing-newline
+Cflags: -I${includedir}/no-trailing-newline
\ No newline at end of file
diff --binary -x .git/ -u -r a/tests/lib1/omg-uninstalled.pc b/tests/lib1/omg-uninstalled.pc
--- a/tests/lib1/omg-uninstalled.pc	2015-07-20 04:16:25.393554107 -0400
+++ b/tests/lib1/omg-uninstalled.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test
+prefix=/usr
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib1/prefix-foo1.pc b/tests/lib1/prefix-foo1.pc
--- a/tests/lib1/prefix-foo1.pc	2015-07-20 04:16:25.393554107 -0400
+++ b/tests/lib1/prefix-foo1.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test/bar
+prefix=/usr/bar
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib1/prefix-foo2.pc b/tests/lib1/prefix-foo2.pc
--- a/tests/lib1/prefix-foo2.pc	2015-07-20 04:16:25.393554107 -0400
+++ b/tests/lib1/prefix-foo2.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test/bar
+prefix=/usr/bar
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib1/quotes.pc b/tests/lib1/quotes.pc
--- a/tests/lib1/quotes.pc	2015-07-20 04:16:25.393554107 -0400
+++ b/tests/lib1/quotes.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test
+prefix=/usr
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib1/static-archive-libs.pc b/tests/lib1/static-archive-libs.pc
--- a/tests/lib1/static-archive-libs.pc	2015-07-20 04:16:25.393554107 -0400
+++ b/tests/lib1/static-archive-libs.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test/local
+prefix=/usr/local
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib1/static-libs.pc b/tests/lib1/static-libs.pc
--- a/tests/lib1/static-libs.pc	2015-07-20 04:16:25.394554607 -0400
+++ b/tests/lib1/static-libs.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test/local
+prefix=/usr/local
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib1/tilde.pc b/tests/lib1/tilde.pc
--- a/tests/lib1/tilde.pc	2015-07-20 04:16:25.394554607 -0400
+++ b/tests/lib1/tilde.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test
+prefix=/usr
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib2/foo.pc b/tests/lib2/foo.pc
--- a/tests/lib2/foo.pc	2015-07-20 04:16:25.394554607 -0400
+++ b/tests/lib2/foo.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test
+prefix=/usr
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/lib3/bar.pc b/tests/lib3/bar.pc
--- a/tests/lib3/bar.pc	2015-07-20 04:16:25.394554607 -0400
+++ b/tests/lib3/bar.pc	2015-07-19 17:33:16.000000000 -0400
@@ -1,4 +1,4 @@
-prefix=/test
+prefix=/usr
 exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include
diff --binary -x .git/ -u -r a/tests/run.sh.in b/tests/run.sh.in
--- a/tests/run.sh.in	2015-07-20 04:16:25.394554607 -0400
+++ b/tests/run.sh.in	2015-07-19 17:33:16.000000000 -0400
@@ -55,13 +55,13 @@
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --libs foo" \
 	'-lfoo'
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --cflags --libs foo" \
-	'-lfoo' '-I/test/include/foo' '-fPIC'
+	'-lfoo' '-I/usr/include/foo' '-fPIC'
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --cflags --libs 'foo > 1.2'" \
-	'-lfoo' '-I/test/include/foo' '-fPIC'
+	'-lfoo' '-I/usr/include/foo' '-fPIC'
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --cflags --libs 'foo > 1.2 bar >= 1.3'" \
-	'-lfoo' '-I/test/include/foo' '-fPIC'
+	'-lfoo' '-I/usr/include/foo' '-fPIC'
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --cflags --libs 'foo' '>' '1.2'" \
-	'-lfoo' '-I/test/include/foo' '-fPIC'
+	'-lfoo' '-I/usr/include/foo' '-fPIC'
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --exists nonexistant; echo \$?" \
 	'1'
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} nonexistant; echo \$?" \
@@ -109,23 +109,23 @@
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --libs bar" \
 	'-lfoo' '-lbar'
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --libs --cflags baz" \
-	'-lbaz' '-fPIC' '-I/test/include/foo'
+	'-lbaz' '-fPIC' '-I/usr/include/foo'
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --static --libs baz" \
 	'-lfoo' '-lbaz' '-lzee'
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --static --libs argv-parse-2" \
 	'-pthread '
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --static --cflags baz" \
-	'-fPIC' '-I/test/include/foo' '-DFOO_STATIC'
+	'-fPIC' '-I/usr/include/foo' '-DFOO_STATIC'
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --static --libs-only-l private-libs-duplication" \
 	'-lprivate -lfoo -lbaz -lzee -lbar -lfoo'
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --static --libs static-libs" \
-	'-lbar -lbar-private -L/test/lib -lfoo'
+	'-lbar -lbar-private -lfoo'
 
 # 4) tests for parser bugs
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --libs dos-lineendings" \
-	'-L/test/lib/dos-lineendings -ldos-lineendings'
+	'-L/usr/lib/dos-lineendings -ldos-lineendings'
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --cflags no-trailing-newline" \
-	'-I/test/include/no-trailing-newline'
+	'-I/usr/include/no-trailing-newline'
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --libs argv-parse" \
 	'-llib-2 -lpthread'
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --exists -foo; echo \$?" \
@@ -141,15 +141,15 @@
 
 # 5) tests for other regressions
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --variable=includedir foo" \
-	'/test/include'
+	'/usr/include'
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --libs-only-L --keep-system-libs cflags-libs-only" \
-	'-L/test/local/lib'
+	'-L/usr/local/lib'
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --libs cflags-libs-only" \
-	'-L/test/local/lib -lfoo'
+	'-L/usr/local/lib -lfoo'
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --libs-only-L --libs-only-l cflags-libs-only" \
-	'-L/test/local/lib -lfoo'
+	'-L/usr/local/lib -lfoo'
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --cflags-only-I --cflags-only-other cflags-libs-only" \
-	'-I/test/local/include'
+	'-I/usr/local/include'
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --libs incomplete; echo $?" \
 	'0'
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --cflags incomplete" \
@@ -161,19 +161,19 @@
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --modversion pkg-config" \
 	'0.28'
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --variable=prefix foo" \
+	'/usr'
+run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --define-variable=prefix=/test --variable=prefix foo" \
 	'/test'
-run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --define-variable=prefix=/test2 --variable=prefix foo" \
-	'/test2'
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --variable=pcfiledir foo" \
 	"${selfdir}/lib1"
 
 # 7) tests for env modifiers
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' PKG_CONFIG_SYSROOT_DIR='${SYSROOT_DIR}' ${1} --cflags baz" \
-	"-I${SYSROOT_DIR}/test/include/foo"
+	"-I${SYSROOT_DIR}/usr/include/foo"
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' PKG_CONFIG_SYSROOT_DIR='${SYSROOT_DIR}' ${1} --variable=prefix baz" \
-	"${SYSROOT_DIR}/test"
+	"${SYSROOT_DIR}/usr"
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' PKG_CONFIG_SYSROOT_DIR='${SYSROOT_DIR}' ${1} --variable=includedir baz" \
-	"${SYSROOT_DIR}/test/include"
+	"${SYSROOT_DIR}/usr/include"
 
 # 8) tests for 'Conflicts'
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --libs conflicts; echo \$?" \
@@ -234,11 +234,11 @@
 
 # test 47 - framework flags
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --libs framework-1" \
-	"-F/test/lib -framework framework-1"
+	"-F/usr/lib -framework framework-1"
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --libs framework-2" \
-	"-F/test/lib -framework framework-2 -framework framework-1"
+	"-F/usr/lib -framework framework-2 -framework framework-1"
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --libs framework-1 framework-2" \
-	"-F/test/lib -framework framework-1 -framework framework-2"
+	"-F/usr/lib -framework framework-1 -framework framework-2"
 
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --exists --print-errors 'foo > 0.6.0 foo < 0.8.0'; echo \$?" \
 	'1'
@@ -247,7 +247,7 @@
 	'1'
 
 run_test "PKG_CONFIG_PATH='${selfdir}/lib2' ${1} ${selfdir}/lib3/bar.pc --cflags" \
-	'-fPIC -I/test/include/foo'
+	'-fPIC -I/usr/include/foo'
 
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --variable=foo case-sensitivity" \
 	"3"
@@ -265,20 +265,20 @@
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' PKG_CONFIG_SYSROOT_DIR='/sysroot2' ${1} --libs sysroot-dir" \
 	'-L/sysroot2/sysroot/lib -lfoo'
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --libs prefix-foo1" \
-	'-L/test/bar/lib -lfoo1'
+	'-L/usr/bar/lib -lfoo1'
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --libs prefix-foo1 prefix-foo2" \
-	'-L/test/bar/lib -lfoo1 -lfoo2'
+	'-L/usr/bar/lib -lfoo1 -lfoo2'
 run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --cflags prefix-foo1 prefix-foo2" \
-	'-I/test/bar/include/foo -DBAR -fPIC -DFOO'
+	'-I/usr/bar/include/foo -DBAR -fPIC -DFOO'
 
 # 10) tests for internal getopt implementation with options at the end
 if [ "x@STRICT_MODE@" = "xno" ]; then
 	run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} foo --libs" \
 		'-lfoo'
 	run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} foo --cflags --libs" \
-		'-lfoo' '-I/test/include/foo' '-fPIC'
+		'-lfoo' '-I/usr/include/foo' '-fPIC'
 	run_test "PKG_CONFIG_PATH='${selfdir}/lib1' ${1} --cflags foo --libs" \
-		'-lfoo' '-I/test/include/foo' '-fPIC'
+		'-lfoo' '-I/usr/include/foo' '-fPIC'
 fi
 
 run_test "${1} --blah; echo \$?" \
